ext/bg/js/japanese.js

```diff
@@ -469,12 +382,7 @@ const jp = (() => {
 
     // Exports
 
-    return {
-        isCodePointKanji,
-        isCodePointKana,
-        isCodePointJapanese,
-        isStringEntirelyKana,
-        isStringPartiallyJapanese,
+    Object.assign(jp, {
```

Comment:
I wonder if this and the reusage of `mixed` `jp` functions are too much complexity just to keep some of the functions out of frontend. I can see a problem with `wanakana` being undefined though, and adding it to frontend html files just to make the unused functions usable seems pointless.

Reply:
Yeah, it's a bit ugly, but unfortunately as you mentioned in a comment below, there's no good way to do modules. And having the object name be just `jp` is convenient; moreso than something like `jpFrontend` and `jpCommon` or something like that.